assets/javascripts/kitten/components/typography/text/index.js

```diff
@@ -94,15 +111,7 @@ Text.propTypes = {
   decoration: PropTypes.oneOf(['underline', 'none']),
   setting: PropTypes.oneOf(['tnum']),
   lineHeight: PropTypes.oneOf(['normal', '1', '1.25', '1.3', '1.4']),
-  size: PropTypes.oneOf([
-    'giant',
-    'huge',
-    'big',
-    'default',
-    'tiny',
-    'micro',
-    'nano',
-  ]),
+  size: PropTypes.oneOf([...textSizes, ...checkDeprecatedSizes]),
```

Comment:
On ne veut accepter que les tailles non dépréciées

```suggestion
  size: PropTypes.oneOf(textSizes),
```